modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomValidation.testcase

```diff
@@ -76,12 +76,69 @@ definition {
 	}
 
 	@description = "LPS-144474 - Verify that, when editing an entry in an Object, the user only is able to update the entry successfully if it passes for all validations from that Object"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test CanUpdateWhenPassesForAllValidations {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object 153558",
+			objectName = "CustomObject153558",
+			pluralLabelName = "Custom Objects 153558");
+
+		ObjectAdmin.addObjectFieldViaAPI(
+			fieldBusinessType = "Text",
+			fieldLabelName = "Custom Field",
+			fieldName = "customField",
+			fieldType = "String",
+			isRequired = "false",
+			objectName = "CustomObject153558");
+
+		ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject153558");
+
+		ObjectAdmin.openObjectAdmin();
+
+		ObjectPortlet.selectCustomObject(label = "Custom Object 153558");
+
+		ObjectAdmin.goToValidationsTab();
+
+		ObjectCustomValidation.addObjectValidation(
+			validationName = "Custom Validation",
+			validationType = "Groovy");
+
+		ObjectCustomValidation.gotoCustomValidation(validationName = "Custom Validation");
+
+		ObjectCustomValidation.activeValidation();
+
+		ObjectCustomValidation.gotoConditionsTab();
+
+		ObjectCustomValidation.insertGroovyScript(groovyScript = "invalidFields = (customField == 'Decline Entry')");
+
+		ObjectCustomValidation.addErrorMessage(errorMessage = "Error: This entry is not possible.");
+
+		Button.clickSave();
+
+		Navigator.openURL();
+
+		ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+			fieldName = "customField",
+			objectName = "CustomObject153558",
+			value = "Entry Test");
+
+		ObjectAdmin.goToCustomObject(objectName = "CustomObject153558");
```

Comment:
See https://github.com/timpak/liferay-portal/pull/252/commits/8af45bf753ce96e11b84a89a6d96b62032218651

There's no need to navigate to the home page because the subsequent macros don't rely on it. ObjectAdmin.addObjectSingleFieldEntryViaAPI calls an API and ObjectAdmin.goToCustomObject goes to a specific URL so we can run those macros from anywhere in portal.